cmd/server/main.go

```
@@ -101,37 +107,59 @@
+		app.logger.DebugContext(ctx, "starting server", slog.String("address", srv.Addr))
+		if err := srv.ListenAndServe(); err != nil && err != http.ErrServerClosed {
+			app.logger.ErrorContext(ctx, "server error", helpers.ErrAttr(err))
+		}
+	}()
+
+	// Wait for termination signal
+	sig := <-signalCh
+	app.logger.InfoContext(ctx, "received signal, shutting down server", slog.String("signal", sig.String()))
+
+	// Shutdown the server gracefully
+	shutdownCtx, shutdownCancel := context.WithTimeout(ctx, 10*time.Second)
```

golangci-lint (1.62.2)

[error] Magic number: 10, in <argument> detected

(mnd)